eclcli/sss/v1/contract.py

```diff
@@ -2,9 +2,7 @@
 
 import json
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    import json
except ImportError:
    import simplejson as json
```